server2/Arcadia.Assistant/Arcadia.Assistant.SF/Scripts/PublishCloud-NewServiceFabricApplication.ps1

```diff
@@ -0,0 +1,344 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
I assume these scripts are supposed to be here, but just in case, I write this comment

Reply:
Yeap. I prefer to add those files in currect PR for avoid separate PR.